graphein/protein/edges/distance.py

```diff
@@ -361,32 +383,11 @@ def add_hydrogen_bond_interactions(
     ]
     """Add all hydrogen-bond interactions."""
     # For these atoms, find those that are within 3.5A of one another.
-    if rgroup_df is None:
-        rgroup_df = G.graph["rgroup_df"]
+    # if rgroup_df is None:
+    #     rgroup_df = G.graph["rgroup_df"]
```

Comment:
What is the purpose of not using the supplied `rgroup_df`?

Reply:
Yes, you are right, I was just debugging the code to determine the hydrogen bonds on the main chain.

Reply:
Yes, you're right. I commented out the code like that just to facilitate debugging and determine the hydrogen bonds on the main chain.